src/test/java/com/arpnetworking/utility/RegexAndMapBenchmarkTest.java

```diff
@@ -0,0 +1,51 @@
+/**
+ * Copyright 2018 Inscope Metrics, Inc
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.arpnetworking.utility;
+
+import com.carrotsearch.junitbenchmarks.BenchmarkOptions;
+import com.carrotsearch.junitbenchmarks.BenchmarkRule;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.TestRule;
+
+import java.util.Collections;
+import java.util.regex.Pattern;
+
+/**
+ * Benchmark tests comparing the RegexAndMapReplacer class against the built-in regex replacement.
+ *
+ * @author Brandon Arp (brandon dot arp at inscopemetrics dot com)
+ */
+public class RegexAndMapBenchmarkTest {
```

Comment:
- `final`
- Not sure this should be run with regular unit tests; at Q we separate PT into E2E-PT and PT

Reply:
But very happy to see this.

Reply:
Eh, it only takes a few seconds to run, so I'm not too concerned.  Lets talk offline about how to add better perf tests.